cabal.project

```diff
@@ -19,15 +19,6 @@ allow-newer:
   , rewrite-inspector:hashable
   , brick:base
 
-repository head.hackage.ghc.haskell.org
```

Comment:
We sadly need this for building locally with GHC 8.8.1 though

Reply:
Just for `vty`/`rewrite-inspector` right?

Reply:
I guess?

Reply:
Hopefully fixed in https://github.com/omelkonian/rewrite-inspector/pull/3 :-)